osquery/logger/plugins/windows_event_log_manifest/readme.txt

```diff
@@ -0,0 +1,5 @@
+The manifest file (osquery.man) can be easily modified using the Manifest Generator (ecmangen.exe) tool found in the Windows SDK. Once it has been edited, you have to compile it using the Message Compiler (mc.exe). Since we are embedding the generated resource script in the main .rc file, it is important to edit it manually to correct the paths.
```

Comment:
Would you mind taking this content and moving it into the `docs\wiki\installation\install-windows.md` documentation? We could hash out a new section detailing how to configure osquery speaking to the Windows Event Logger?